src/terminal/adapter/adaptDispatch.hpp

```diff
@@ -148,11 +156,12 @@ namespace Microsoft::Console::VirtualTerminal
         std::unique_ptr<AdaptDefaults> _pDefaults;
         TerminalOutput _TermOutput;
 
-        COORD _coordSavedCursor;
+        CursorState _savedCursorState[2];
```

Comment:
Since it's just implicitly the true/false for the non-alternate and alternate.... can you leave a comment here describing the two positions in the array? ([0] is used for regular, [1] is used for alternate) so folks reading later understand why 2 and what they're for.

Reply:
Comment added in commit 5bc16a28623e08fe8d43a051efc27c29182cdf10. That also includes updates to the other comments you pointed out.